impl/windows/windows.js

```diff
@@ -50,15 +66,15 @@ var setVolume = function(val, done) {
 var getMuted = function(done) {
 	done = done || function () {};
 
-	getVolume(function(err, volume) {
+	getVolumeInfo((err, info) => {
```

Comment:
Arrow functions doesn't work in node 0.x, let's wait a little bit with breaking compatibility :)